src/colors/conversions.jl

```diff
@@ -0,0 +1,43 @@
+"""
+    channelify(m::AbstractArray{<:Color})
+Channelify function.
+
+Input should be in ``WHCN`` order ``\\implies (*, batch)``.
+Output will be in the order ``(*, channels, batch)``.
+"""
+function channelify(m::AbstractArray{T}) where T <: Color
+    m = channelview(m)
+    t = collect(1:ndims(m))
```

Comment:
use `Tuple` whenever possible

```suggestion
    t = ntuple(_->i, ndims(m))
```